src/components/GrampsjsFormSelectType.js

```diff
@@ -85,9 +87,47 @@ class GrampsjsFormSelectType extends GrampsjsTranslateMixin(LitElement) {
               )}
         </mwc-select>
       </p>
+
+      <mwc-button
+        @click="${this.switchTypeInput}"
+      >
+        Enter custom type instead
+      </mwc-button>
+
+      ${this._hasCustomType === false
+        ? html``
+        : html`
+          <h4 class="label">${this.heading || this._('Custom Type')}</h4>
```

Comment:
`this.heading` is a way to override the default `Type` label of the type selector. It doesn't make sense to override "Custom Type` with the same heading, does it?

Reply:
This was just a misunderstanding of the architecture. I thought that `${this.heading || this._('Custom Type')}` was required in the event a heading was not able to generate. I agree it is not necessary. I've changed it to 
`${this._('Custom Type')}`. 

_Note: I am also adding "Custom Type" to en.json_